tests-legacy/Integration/classes/LanguageTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace LegacyTests\Integration\classes;
+
+use LegacyTests\PrestaShopBundle\Utils\DatabaseCreator as Database;
+use LegacyTests\TestCase\IntegrationTestCase;
+use Language;
+
+class LanguageTest extends IntegrationTestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+
+        Database::restoreTestDB();
+    }
+
+    public function testGetIdByIsoWithCache()
+    {
+        $this->assertNull(Language::getIdByIso('zz'));
+        $this->addDummyLanguage();
+        $this->assertIsString(Language::getIdByIso('zz'));
```

Comment:
It returns a string? Not an `int` ?
Besides even if it's not strictly typed, it would be better to add a test to ensure it's not 0 like in currency test

Reply:
Fixed ;)